src/SourceBuild/content/eng/pipelines/pr.yml

```diff
@@ -51,3 +51,8 @@ stages:
       scope: lite
     ${{ else }}:
       scope: full
+
+- template: /src/installer/eng/pipelines/templates/stages/vmr-final-join.yml@self
+  parameters:
+    azureDevOpsOrg: dnceng-public
+    azureDevOpsProject: public
```

Comment:
It feels to me that these values should be stored in a variables file instead and be condition on public vs non-public.

Reply:
yeah, fair point, done